File: C#/1/1_3.cs
using System;

class people
{
    private string name;
    private int age;

    public people(string n, int a)
    {
        this.name = n;
        this.age = a;
    }

    public void printed()
    {
        Console.WriteLine(this.name + " is " + this.age + " years old.");
    }
}

class Program
{
    static void Main()
    {
        people person = new people("Alice", 30);
        person.printed();
    }
}

File: JavaScript/JavaScript/1/1_3.js
class constructorOfPerson {
  constructor(person, year) {
      this.person = person;
      this.year = this.year;
  }

  extractToConsole() {
      console.log(this.person + " is " + this.year + " years old.");
  }
}

let finalResult = new constructorOfPerson("Alice", 30);
finalResult.extractToConsole();
